.github/workflows/integration_test.yml

```diff
@@ -0,0 +1,70 @@
+name: Integration Tests
+on: [pull_request]
+
+jobs:
+  tutor-integration-test:
+    name: Integration with Tutor
+    strategy:
+      matrix:
+        tutor_version: ["<19.0.0", "<18.0.0", "<17.0.0", "<16.0.0"]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Prepare Tutor & launch
+        run: |
+          pip install "tutor${{ matrix.tutor_version }}"
+          pip install -e .
+          TUTOR_ROOT="$(pwd)" tutor --version
+          TUTOR_ROOT="$(pwd)" tutor config save
+
+      - name: Set up SSH
+        run: |
+          mkdir -p ~/.ssh
+          echo "${{ secrets.SSH_PRIVATE_KEY }}" > ~/.ssh/id_rsa
+          chmod 600 ~/.ssh/id_rsa
+          ssh-keyscan github.com >> ~/.ssh/known_hosts
+
+      - name: Enable the tutor plugin
+        run: |
+          TUTOR_ROOT="$(pwd)" tutor plugins enable picasso
+          TUTOR_ROOT="$(pwd)" tutor picasso -h
+
+      - name: Adding data to the config.yml
+        run: |
+          echo "PICASSO_MANAGE_DPKG:" | tee -a ./config.yml
+          echo "  name: eox-manage" | tee -a ./config.yml
+          echo "  repo: git@github.com:eduNEXT/eox-manage.git" | tee -a ./config.yml
+          echo "  version: v5.2.0" | tee -a ./config.yml
+          echo "PICASSO_THEMES:" | tee -a ./config.yml
+          echo "- name: endx-saas-themes" | tee -a ./config.yml
+          echo "  repo: git@github.com:eduNEXT/ednx-saas-themes.git" | tee -a ./config.yml
+          echo "  version: master" | tee -a ./config.yml
+          echo "PICASSO_EXTRA_COMMANDS:" | tee -a ./config.yml
+          echo "- tutor plugins update" | tee -a ./config.yml
+          echo "- tutor plugins index add https://raw.githubusercontent.com/eduNEXT/tutor-plugin-indexes/picasso_test/" | tee -a ./config.yml
+          echo "- tutor plugins install mfe mfe_extensions aspects" | tee -a ./config.yml
+          echo "- tutor plugins enable mfe mfe_extensions aspects" | tee -a ./config.yml
+          echo "- tutor config save" | tee -a ./config.yml
```

Comment:
maybe the `echo` and the `tee` commands can be simplified:

``` bash
cat <<EOF >> config.yml
PICASSO_MANAGE_DPKG:
  name: eox-manage
  repo: git@github.com:eduNEXT/eox-manage.git
  version: v5.2.0
PICASSO_THEMES:
  - name: endx-saas-themes
    repo: git@github.com:eduNEXT/ednx-saas-themes.git
    version: master
PICASSO_EXTRA_COMMANDS:
  - tutor plugins update
  - tutor plugins index add https://raw.githubusercontent.com/eduNEXT/tutor-plugin-indexes/picasso_test/
  - tutor plugins install mfe mfe_extensions aspects
  - tutor plugins enable mfe mfe_extensions aspects
  - tutor config save
EOF
```

Reply:
Thanks for this ✨